omrmakefiles/rules.linux.mk

```diff
@@ -58,19 +59,19 @@ ifeq (ppc,$(OMR_HOST_ARCH))
     endif
 endif
 
-# Compile without exceptions
+## Compile without exceptions
 ifeq (gcc,$(OMR_TOOLCHAIN))
-ifeq (1,$(OMR_RTTI))
-    GLOBAL_CXXFLAGS+=-fno-exceptions -fno-threadsafe-statics
-else
-    GLOBAL_CXXFLAGS+=-fno-exceptions -fno-rtti -fno-threadsafe-statics
-endif
-else
-ifeq (1,$(OMR_RTTI))
-    GLOBAL_CXXFLAGS+=-qrtti -qnoeh
+    ifeq (1,$(OMR_RTTI))
```

Comment:
It's best to fight the urge to make formatting improvements that don't directly relate to the functional change that the PR introduces, because it makes life hell for your reviewers, who have to spend a lot of time trying to figure out if a particular diff is functional or not. Prettifying unrelated code is best done in it's own PR so that reviewers know what to expect.

Reply:
Yeah fair enough; I believe the IDE that @jeanlego and I are using does some auto whitespace stripping/formatting.

Reply:
No this was me. I've rearranged things so that's its clearer where to put new architecture in. (Change if/else ladders to if else elif format)

Reply:
Should be cleaned up.